src/main/java/edu/virginia/lib/fedora/aview/AView.java

```diff
@@ -42,31 +47,50 @@ public AView() throws URISyntaxException, MalformedURLException {
         accessionsTemplate = ve.getTemplate("accessions.vm");
         bagsTemplate = ve.getTemplate("bags.vm");
         filesTemplate = ve.getTemplate("files.vm");
+        
+        try {
+        	BufferedReader br = new BufferedReader(new FileReader("host.txt"));
```

Comment:
Try FileUtils.readFiletoString() to save some typing.

This will need to be added to your pom.xml file.
    <dependency>
      <groupId>commons-io</groupId>
      <artifactId>commons-io</artifactId>
      <version>2.5</version>
    </dependency>